app/components/options.vue

```diff
@@ -6,7 +6,7 @@
       <ul>
         <li>
           <h2>Choose a language</h2>
-          <select id="language" v-model="language" @change="onLanguageSelect">
+          <select id="language" :value="language" @change="onLanguageSelect">
```

Comment:
Yeah so it's due to us using Vuex and `language` thus being read from the store as a computed property. V-model has reactive data binding, so changing the model in the view would try to directly mutate `language`, but we want all updates to be done via an action or mutation so it gets updated in the app's store. Binding `language` to the input value, removes the two-way data binding, and simplifies sending the new value to the store via the mutation called in the onChange callback (if that makes sense?)